tools/sigclient/cmd/sigclient.go

```diff
@@ -99,6 +100,41 @@ var esBulkCmd = &cobra.Command{
 	},
 }
 
+var functionalTestCmd = &cobra.Command{
+	Use:   "functional",
+	Short: "functional testing of SigLens",
+	Run: func(cmd *cobra.Command, args []string) {
+		processCount, _ := cmd.Flags().GetInt("processCount")
```

Comment:
don't expose `processCount` , `totalEvents`, `numIndices`, `indexPrefix`, `indexName`, `batchSize`. 
a user should not be able to modify it on cmdline. This should be inititialized to fixed consts in the code

Reply:
addressed in the latest commit.